### ISSUE TITLE
Centos 7 Error: Package 

### ISSUE THREAD

**Issue by forkeer** [rabbitmq/rabbitmq-server#1263]
Hi
When need update Erlang from yum show this error :

```
Error: Package: erlang-gs-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-kernel(x86-64) = 19.3-1.el7.centos
           Removing: erlang-kernel-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-kernel(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-kernel-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-kernel-R16B-03.18.el7.x86_64 (epel)
               erlang-kernel(x86-64) = R16B-03.18.el7
           Available: erlang-kernel-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-kernel-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.1-3.el7.centos
           Available: erlang-kernel-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.3-1.el7.centos
           Available: erlang-kernel-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.4-1.el7.centos
           Available: erlang-kernel-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.5-1.el7.centos
           Available: erlang-kernel-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-kernel-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.0-1.el7.centos
           Available: erlang-kernel-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.1-1.el7.centos
           Available: erlang-kernel-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.2-1.el7.centos
           Available: erlang-kernel-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.3-1.el7.centos
           Available: erlang-kernel-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.0-1.el7.centos
           Available: erlang-kernel-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.1-1.el7.centos
           Available: erlang-kernel-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.2-1.el7.centos
           Available: erlang-kernel-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-percept-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-erts(x86-64) = 19.3-1.el7.centos
           Removing: erlang-erts-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-erts(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-erts-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-erts-R16B-03.18.el7.x86_64 (epel)
               erlang-erts(x86-64) = R16B-03.18.el7
           Available: erlang-erts-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-erts-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.1-3.el7.centos
           Available: erlang-erts-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.3-1.el7.centos
           Available: erlang-erts-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.4-1.el7.centos
           Available: erlang-erts-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.5-1.el7.centos
           Available: erlang-erts-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-erts-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.0-1.el7.centos
           Available: erlang-erts-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.1-1.el7.centos
           Available: erlang-erts-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.2-1.el7.centos
           Available: erlang-erts-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.3-1.el7.centos
           Available: erlang-erts-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.0-1.el7.centos
           Available: erlang-erts-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.1-1.el7.centos
           Available: erlang-erts-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.2-1.el7.centos
           Available: erlang-erts-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-percept-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-inets(x86-64) = 19.3-1.el7.centos
           Removing: erlang-inets-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-inets(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-inets-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-inets-R16B-03.18.el7.x86_64 (epel)
               erlang-inets(x86-64) = R16B-03.18.el7
           Available: erlang-inets-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-inets-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.1-3.el7.centos
           Available: erlang-inets-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.3-1.el7.centos
           Available: erlang-inets-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.4-1.el7.centos
           Available: erlang-inets-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.5-1.el7.centos
           Available: erlang-inets-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-inets-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 18.0-1.el7.centos
           Available: erlang-inets-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 18.1-1.el7.centos
           Available: erlang-inets-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 18.2-1.el7.centos
           Available: erlang-inets-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 18.3-1.el7.centos
           Available: erlang-inets-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 19.0-1.el7.centos
           Available: erlang-inets-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 19.1-1.el7.centos
           Available: erlang-inets-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 19.2-1.el7.centos
           Available: erlang-inets-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-inets(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-percept-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-stdlib(x86-64) = 19.3-1.el7.centos
           Removing: erlang-stdlib-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-stdlib(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-stdlib-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-stdlib-R16B-03.18.el7.x86_64 (epel)
               erlang-stdlib(x86-64) = R16B-03.18.el7
           Available: erlang-stdlib-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-stdlib-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.1-3.el7.centos
           Available: erlang-stdlib-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.3-1.el7.centos
           Available: erlang-stdlib-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.4-1.el7.centos
           Available: erlang-stdlib-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.5-1.el7.centos
           Available: erlang-stdlib-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-stdlib-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.0-1.el7.centos
           Available: erlang-stdlib-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.1-1.el7.centos
           Available: erlang-stdlib-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.2-1.el7.centos
           Available: erlang-stdlib-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.3-1.el7.centos
           Available: erlang-stdlib-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.0-1.el7.centos
           Available: erlang-stdlib-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.1-1.el7.centos
           Available: erlang-stdlib-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.2-1.el7.centos
           Available: erlang-stdlib-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-gs-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-erts(x86-64) = 19.3-1.el7.centos
           Removing: erlang-erts-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-erts(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-erts-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-erts-R16B-03.18.el7.x86_64 (epel)
               erlang-erts(x86-64) = R16B-03.18.el7
           Available: erlang-erts-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-erts-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.1-3.el7.centos
           Available: erlang-erts-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.3-1.el7.centos
           Available: erlang-erts-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.4-1.el7.centos
           Available: erlang-erts-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.5-1.el7.centos
           Available: erlang-erts-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-erts-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.0-1.el7.centos
           Available: erlang-erts-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.1-1.el7.centos
           Available: erlang-erts-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.2-1.el7.centos
           Available: erlang-erts-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 18.3-1.el7.centos
           Available: erlang-erts-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.0-1.el7.centos
           Available: erlang-erts-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.1-1.el7.centos
           Available: erlang-erts-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 19.2-1.el7.centos
           Available: erlang-erts-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-erts(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-percept-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-runtime_tools(x86-64) = 19.3-1.el7.centos
           Removing: erlang-runtime_tools-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-runtime_tools(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-runtime_tools-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-runtime_tools-R16B-03.18.el7.x86_64 (epel)
               erlang-runtime_tools(x86-64) = R16B-03.18.el7
           Available: erlang-runtime_tools-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-runtime_tools-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.1-3.el7.centos
           Available: erlang-runtime_tools-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.3-1.el7.centos
           Available: erlang-runtime_tools-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.4-1.el7.centos
           Available: erlang-runtime_tools-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.5-1.el7.centos
           Available: erlang-runtime_tools-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-runtime_tools-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 18.0-1.el7.centos
           Available: erlang-runtime_tools-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 18.1-1.el7.centos
           Available: erlang-runtime_tools-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 18.2-1.el7.centos
           Available: erlang-runtime_tools-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 18.3-1.el7.centos
           Available: erlang-runtime_tools-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 19.0-1.el7.centos
           Available: erlang-runtime_tools-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 19.1-1.el7.centos
           Available: erlang-runtime_tools-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 19.2-1.el7.centos
           Available: erlang-runtime_tools-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-runtime_tools(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-percept-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-kernel(x86-64) = 19.3-1.el7.centos
           Removing: erlang-kernel-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-kernel(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-kernel-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-kernel-R16B-03.18.el7.x86_64 (epel)
               erlang-kernel(x86-64) = R16B-03.18.el7
           Available: erlang-kernel-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-kernel-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.1-3.el7.centos
           Available: erlang-kernel-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.3-1.el7.centos
           Available: erlang-kernel-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.4-1.el7.centos
           Available: erlang-kernel-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.5-1.el7.centos
           Available: erlang-kernel-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-kernel-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.0-1.el7.centos
           Available: erlang-kernel-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.1-1.el7.centos
           Available: erlang-kernel-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.2-1.el7.centos
           Available: erlang-kernel-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 18.3-1.el7.centos
           Available: erlang-kernel-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.0-1.el7.centos
           Available: erlang-kernel-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.1-1.el7.centos
           Available: erlang-kernel-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 19.2-1.el7.centos
           Available: erlang-kernel-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-kernel(x86-64) = 20.0rc1-3.el7.centos
Error: Package: erlang-gs-19.3-1.el7.centos.x86_64 (@erlang-solutions)
           Requires: erlang-stdlib(x86-64) = 19.3-1.el7.centos
           Removing: erlang-stdlib-19.3-1.el7.centos.x86_64 (@erlang-solutions)
               erlang-stdlib(x86-64) = 19.3-1.el7.centos
           Updated By: erlang-stdlib-20.0rc2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 20.0rc2-1.el7.centos
           Available: erlang-stdlib-R16B-03.18.el7.x86_64 (epel)
               erlang-stdlib(x86-64) = R16B-03.18.el7
           Available: erlang-stdlib-17.1-1.1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.1-1.1.el7.centos
           Available: erlang-stdlib-17.1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.1-3.el7.centos
           Available: erlang-stdlib-17.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.3-1.el7.centos
           Available: erlang-stdlib-17.4-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.4-1.el7.centos
           Available: erlang-stdlib-17.5-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.5-1.el7.centos
           Available: erlang-stdlib-17.5.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 17.5.3-1.el7.centos
           Available: erlang-stdlib-18.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.0-1.el7.centos
           Available: erlang-stdlib-18.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.1-1.el7.centos
           Available: erlang-stdlib-18.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.2-1.el7.centos
           Available: erlang-stdlib-18.3-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 18.3-1.el7.centos
           Available: erlang-stdlib-19.0-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.0-1.el7.centos
           Available: erlang-stdlib-19.1-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.1-1.el7.centos
           Available: erlang-stdlib-19.2-1.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 19.2-1.el7.centos
           Available: erlang-stdlib-20.0rc1-3.el7.centos.x86_64 (erlang-solutions)
               erlang-stdlib(x86-64) = 20.0rc1-3.el7.centos
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. This assumes two things:

1. GitHub issues are not used for questions, investigations, root cause analysis, discussions of potential issues, etc (as defined by this team)
2. We have a certain amount of information to work with

We get at least a dozen of questions through various venues every single day, often quite light on details.
At that rate GitHub issues can very quickly turn into a something impossible to navigate and make sense of even for our team. Because of that questions, investigations, root cause analysis, discussions of potential features are all considered to be mailing list material by our team. Please post this to [rabbitmq-users](https://groups.google.com/forum/#!forum/rabbitmq-users).

Getting all the details necessary to reproduce an issue, make a conclusion or even form a hypothesis about what's happening can take a fair amount of time. Our team is multiple orders of magnitude smaller than the RabbitMQ community. Please help others help you by providing a way to reproduce the behavior you're
observing, or at least sharing as much relevant information as possible [on the list](https://groups.google.com/forum/#!forum/rabbitmq-users):

 * Server, client library and plugin (if applicable) versions used
 * Server logs
 * A code example or terminal transcript that can be used to reproduce
 * **Full** exception stack traces (not a single line message)
 * `rabbitmqctl status` (and, if possible, `rabbitmqctl environment` output)
 * Other relevant things about the environment and workload, e.g. [a traffic capture](https://www.rabbitmq.com/amqp-wireshark.html)

Feel free to edit out hostnames and other potentially sensitive information.

When/if we have enough details and evidence we'd be happy to file a new issue.

Thank you.

**Comment by michaelklishin** [comment]
The error says that package `erlang-gs-19.3-1.el7.centos.x86_64` requires `erlang-kernel(x86-64) = 19.3-1.el7.centos`. I'm not sure how it has anything to do with RabbitMQ.

While our team does maintain a stripped down [Erlang RPM](https://github.com/rabbitmq/erlang-rpm), the packages in the output seem to come from the Erlang Solutions repository.

**Comment by Farit-Biktimirov** [comment]
@michaelklishin , would you kindly describe workaround for this type of BUG?

**Comment by michaelklishin** [comment]
@Farit-Biktimirov there is no single piece of advice for resolving package dependency hell. The easiest option is to use the Erlang RPM that we build. It has no dependencies (other than a modern OpenSSL).
